arcaflow_plugin_pcp/pcp_plugin.py

```diff
@@ -210,14 +210,14 @@ def post_process(
         pcp2csv_cmd.extend(pcp2_flags)
         pcp2csv_cmd.extend(metrics)
 
-    max_retries = 1
+    retry_limit = 2
     pcp2csv_status = ""
     pcp2json_status = ""
-    # Here we give max_retries chances to run the pcp2json conversion.
+    # Here we give an additional chance to run the pcp2json conversion.
     # This covers the situation where pmlogger is cancelled before a
     # params.pmlogger_interval time has passed, which can cause pcp2json
     # to fail.
-    for _attempt in range(max_retries):
+    for _attempt in range(retry_limit):
```

Comment:
Not to be pedantic or anything, but this code is going to "try" once and "retry" once, so, rather than setting the "retry limit" to two, I recommend one of two approaches:
- either name the variable "tries" instead of "retries"
- or leave it as "retries", set it to `1` instead of `2`, and add one in the `range()` call.

Either approach would be more expressive.

Reply:
Ah, but the limit is exclusive, not inclusive. ;)

Reply:
Perhaps, but I don't feel like either the code or the comment expresses that intention very well.  😞